python/tests/test_logistic_regression.py

```diff
@@ -2376,3 +2377,41 @@ def test_sparse_one_gpu_all_zeroes(
         cpu_lr = SparkLogisticRegression(**params)
         cpu_model = cpu_lr.fit(bdf)
         compare_model(gpu_model, cpu_model, bdf)
+
+
+@pytest.mark.parametrize("gpuMemRatioForData", ["None", "0.6"])
+def test_gpuMemRatioForData(
+    gpuMemRatioForData: Optional[float],
+    gpu_number: int,
+) -> None:
+    gpu_number = min(gpu_number, 2)
+
+    with CleanSparkSession() as spark:
+        assert (
+            _get_spark_session().conf.get("spark.rapids.ml.gpuMemRatioForData", None)
```

Comment:
Would be good to document this in `site/configuration.md`

Reply:
Added docstring in configuration.md.